gslib/tests/testcase/integration_testcase.py

```diff
@@ -527,8 +531,18 @@ def CreateBucket(self,
       location = boto.config.get(
           'GSUtil', 'test_cmd_regional_bucket_location', 'us-central1')
 
+    bucket_name_prefix = six.ensure_text(bucket_name_prefix)
+    bucket_name_suffix = six.ensure_text(bucket_name_suffix)
+
     if bucket_name:
+      bucket_name = ''.join([bucket_name_prefix,
+                             bucket_name,
+                             bucket_name_suffix])
       bucket_name = util.MakeBucketNameValid(bucket_name)
+    else:
+      bucket_name = self.MakeTempName(''.join([bucket_name_prefix,
```

Comment:
I don't think this is semantically correct.  The arg you're passing corresponds to this method's `kind` parameter, so the "prefix" and "suffix" actually gets crammed into the middle of the generated name.  This method (defined in gslib/tests/testcase/base.py) also has a `prefix` parameter -- we should pass the prefix using that.  Similarly, it might be worth adding `suffix` as another kwarg and passing the suffix along that way.

Reply:
Whoops! Sorry, I should have checked the method signature, my bad. :) Will fix